tests/bench_msg_pingpong/tests/01-run.py

```diff
@@ -16,6 +15,5 @@ def testfunc(child):
 
 
 if __name__ == "__main__":
-    sys.path.append(os.path.join(os.environ['RIOTTOOLS'], 'testrunner'))
```

Comment:
The line below can now be moved with the other imports at the top of the script.

Reply:
yes will do